cognite_toolkit/cdf_tk/bootstrap.py

```diff
@@ -321,9 +321,11 @@ def check_auth(
         print("  [bold red]ERROR[/]: Unable to retrieve CDF groups.")
         ToolGlobals.failed = True
         return None
-    read_write = Group.load(
-        Path(f"{Path(__file__).parent.parent.as_posix()}{group_file}").read_text(),
-    )
+    if Path(group_file).exists():
+        file_text = Path(group_file).read_text()
+    else:
+        file_text = Path(f"{Path(__file__).parent.parent.as_posix()}{group_file}").read_text()
```

Comment:
Why are you looking to levels above this file for this group_file? This will not hold when a used do pip install as then it will likely be in some virtual environment somewhere you do not control.